### ISSUE TITLE
Cannot tail server logs

### ISSUE THREAD

**Issue by ArnaudLier** [symfony-cli/symfony-cli#144]
When executing `symfony server:start` without the `-d` switch, I can see the logs correctly. But when running `symfony server:log` in another terminal (or with the server started with the `-d` switch), the only output is:
```
Tailing Web Server log file (/home/arnaud/.symfony5/log/741ad4550b0018165e0552aeeb2c5405c7961120.log)
Tailing PHP log file (/home/arnaud/.symfony5/log/741ad4550b0018165e0552aeeb2c5405c7961120/7daf403c7589f4927632ed3b6af762a992f09b78.log)
```
I can manually tail the said files but it's not very convenient.

**Comment by tucksaun** [comment]
I'm not sure to understand accurately, so let me rephrase: you would like the server:log command to output the N last lines of the log right?

When doing the initial implementation with @fabpot we decided to only follow new logs by default because we sensed if would be more natural for most users in development mode: you start to follow the logs and then hit the browser playing you scenarios and only your current logs are shown.
But if you want, you can use the `--lines`( or `-n`) option to also tail previous N lines before following the logs.

Does that answer your question?

**Comment by ArnaudLier** [comment]
> I'm not sure to understand accurately, so let me rephrase: you would like the server:log command to output the N last lines of the log right?
> 
> When doing the initial implementation with @fabpot we decided to only follow new logs by default because we sensed if would be more natural for most users in development mode: you start to follow the logs and then hit the browser playing you scenarios and only your current logs are shown. But if you want, you can use the `--lines`( or `-n`) option to also tail previous N lines before following the logs.
> 
> Does that answer your question?

Yes, thank you! Sorry, didn't see that `--lines` option!

**Comment by tucksaun** [comment]
Great!

But I see where the wording can be confusing... see #145 for a fix
